### ISSUE TITLE
i have a quesion

### ISSUE THREAD

**Issue by lmk123568** [ultralytics/yolov5#3049]
## ❔Question
I have modified the model and plan to retrain COCO data，GPU is a GTX2070，batchsize = 8 
Do I need to modify the learning rate？such as lr0/5 ?Or is there anything else you should pay attention to

**Comment by glenn-jocher** [comment]
@lmk123568 no you don't need to modify anything. LR scales automatically with batch size. For more info you may want to see our batch-size study https://github.com/ultralytics/yolov5/discussions/2452

**Comment by lmk123568** [comment]
> 
> 
> @lmk123568 no you don't need to modify anything. LR scales automatically with batch size. For more info you may want to see our batch-size study #2452

wow~ thank！

**Comment by lmk123568** [comment]
> 
> 
> @lmk123568 no you don't need to modify anything. LR scales automatically with batch size. For more info you may want to see our batch-size study #2452


Thank you very much for your reply！！I want to know how to start training VOC datasets from scratch without the need for a pre-training model，How should the parameters be set?Use hyp.finetune directly?

**Comment by glenn-jocher** [comment]
@lmk123568 the Colab notebook includes VOC training commands in the Appendix section, you might want to start there.
https://colab.research.google.com/github/ultralytics/yolov5/blob/master/tutorial.ipynb

**Comment by lmk123568** [comment]
> @lmk123568 the Colab notebook includes VOC training commands in the Appendix section, you might want to start there.
> https://colab.research.google.com/github/ultralytics/yolov5/blob/master/tutorial.ipynb

ok，thank u
